scala-package/macros/src/test/scala/org/apache/mxnet/MacrosSuite.scala

```diff
@@ -36,7 +36,7 @@ class MacrosSuite extends FunSuite with BeforeAndAfterAll {
     )
     val output = List(
       ("org.apache.mxnet.Symbol", true),
-      ("Int", false),
```

Comment:
wouldn't this break backward compatibility?

Reply:
This test is just for Macros generation. As long as integration test passed, it didn't break the BC.